claripy/strings.py

```diff
@@ -39,12 +42,13 @@ def StrExtract(high, low, str_val):
 def StrReplace(initial_string, pattern_to_be_replaced, replacement_pattern):
     """
     Create a concrete version of the replaced string
-    (replace ONLY th efirst occurrence of the pattern)
+    (replace ONLY the first occurrence of the pattern)
 
     :param initial_string: string in which the pattern needs to be replaced
-    :param pattern_to_be_replaced: substring that has to be replaced inside initial_string
-    :param replacement_pattern: pattern that has to be inserted in initial_string to replace
-                                 pattern_to_be_replaced
+    :param pattern_to_be_replaced: substring that has to be replaced inside
+                                   initial_string
+    :param replacement_pattern: pattern that has to be inserted in
+                                initial_string to replace pattern_to_be_replaced
```

Comment:
The way we usually do indentation in docstrings is that all the text after the names gets indented to the same level across all the lines. If that's too obnoxious you can stick a newline after the `:param name:`.

Also, we prefer to wrap our text at 100 cols in this project.